Assignment2/InviteFriends.cpp

```diff
@@ -0,0 +1,67 @@
+#include "InviteFriends.h"
+
+using namespace std;
+
+double InviteFriends::getDistance(const double latitude, const double longitude)
```

Comment:
this problem was perfect for using Object oriented programming. You could have defined a LatLng class that encapsulats distance function. Then a separate Friend class and Party class could have done rest of the work.

Reply:
Do you mean to say that for each Friend object we store a separate LatLng object and LatLng class will calculate the distance and may contain some other info also and the Party class (currently InviteFriends) would just check and store the list of Friend objects ?

Reply:
LatLng should store the lat lng, thats why it is named LatLng class.